rustfmt.toml

```diff
@@ -18,7 +17,7 @@ ignore = [
     # but we still want to format rmake.rs files in tests/run-make/ so we need to do this
     # dance to avoid the parent directory from being excluded
     "!/tests/run-make/",
-    "/tests/run-make/*/*.rs",
+    "/tests/run-make/*",
```

Comment:
This change doesn't work for me locally.

I changed it to yours, than changed the formatting inside a `rmake.rs` and than did `./x.py fmt` and while the "formatting modified file ..." message was print, the file wasn't formatted, without this change it is correctly formatted.

Reply:
Yeah, the behaviour of these `rmake.rs` rules is really weird. 
- I was seeing the original problem when I was explicitly listing files to format.
- If you change the style in `rustfmt.toml` majorly (e.g. add `imports_granularity = "Item"`) and then run `x fmt`, it should do nothing, because no files have changed. But it formats all the `rmake.rs` files.

Reply:
In this case, it might be simpler to update how the rule is implemented in bootstrap to make it more straightforward I think.